tests_web/conftest.py

```diff
@@ -92,12 +92,13 @@ def create_auth(username, password):
         }
 
     @classmethod
-    def monkeypatch_betamaxerror(cls):
+    def monkeypatch_betamaxerror(cls, monkeypatch):
+        cls.BETAMAX_ERRORS = 0
```

Comment:
Já mám pořád pocit, že tady prostě by to zasloužilo buďto nulovat po tom assertu v try/finally, nebo to monkeypatchem nastavit jako atribut třídy:

```python
monkeypatch.setattr(betamax.BetamaxError, 'errors', 0, raising=False)
```

Ale možná to takhle nějak funguje, opravdu střílím od boku, kód jsem nezkoušel.

Reply:
Teď to funguje tak, že když vyjímku chytím, tak to skončí na **E**rror v teardown (ten assert). Další testy už ale projdou v pohodě, tak jak mají (správně). Pokud vyjímku nechytím, tak test **F**ailne a pak je ještě ten **E**rror navíc. Pokud je všechno správně (žádný `BetamaxError` se nevytvoří), tak to projde... Přidání do finally se nic nezměnilo, ale je to opravdu asi čistější, než nulovat jenom před monkeypatch.

Co by mělo dělat `errors`?

Reply:
Počítat si to na té třídě. Ale to je asi overkill.